### ISSUE TITLE
ProGen v2.0.2 profiles no longer work in S3D 4.0

### ISSUE THREAD

**Issue by hst68** [BCN3D/BCN3DSigma-Profile-Generator#8]
The S3D ProGen Profiles seem to me to be a bit flawed. See screenshots.
I put together the following compilation on ProGen website:
Left Extruder 0.3
Right Extruder 0.5
In S3D I only get the Left Extruder.

other configurations, e. g. only right-hand extruders, do not work. Also the Nozzles diameters are not copied correctly.
The ProGens from last week are all working.

I have now downloaded several new profiles. And none of them works in S3D. But if I take the previously created settings, all settings will be taken over from the ProGen profile.

![screenshot at okt 31 20-30-23](https://user-images.githubusercontent.com/32851794/32294316-0d476a52-bf46-11e7-82e1-6250fb27d65b.png)
![screenshot at okt 31 20-28-33](https://user-images.githubusercontent.com/32851794/32294317-0d627bc6-bf46-11e7-82f3-f4bbcce861c2.png)

**Comment by guillemap** [comment]
Hello, all profiles are working for us. Did you try to remove the old imported ones and import the new ones again?

**Comment by thesec** [comment]
@hst68 S3D is flawed itself. Use a working and actively supported slicer.

**Comment by hst68** [comment]
The problem was that the profiles of S3D were not deleted correctly from the database. 
They weren't displayed anymore. But with the new add, all profiles didn't work anymore. Even if you uninstall S3D, the. plist will remain in the library.
Only when I have deleted the following entries via terminal. I was able to install S3D cleanly again and read the FFF profiles correctly!

For completeness if you still want to do a complete uninstall:
[*] Run the built in uninstall and choose to remove the license. I like to play it safe and not have a license tied up on a non-operational machine. Although there is an option to reclaim the license online for the current version should you forget.
[*] Run the following 2 commands in terminal.
[*] rm -rf /Users/$USER/Library/Application Support/Simplify3D
[*] rm /Users/$USER/Library/Preferences/com.simplify3d.S3D-Software.plist
